modules/apps/transfer/keeper/relay.go

```diff
@@ -59,18 +59,18 @@ func (k Keeper) SendTransfer(
 	receiver string,
 	timeoutHeight clienttypes.Height,
 	timeoutTimestamp uint64,
-) error {
+) (uint64, error) {
```

Comment:
Could we mention in the godoc of this function something like `The first return value is the sequence number of the packet sent. It will be greater than zero in case of success, zero otherwise.`?

Reply:
Good catch, updated.